frontend/src/components/Gallery/AppBar/Platform/PlatformInfoDrawer.vue

```diff
@@ -49,13 +49,32 @@ const aspectRatioOptions = computed(() => [
 ]);
 
 const platformInfoFields = [
+  { key: "name", label: "Name" },
   { key: "slug", label: "Slug" },
   { key: "fs_slug", label: t("platform.filesystem-folder-name") },
   { key: "category", label: t("platform.category") },
   { key: "generation", label: t("platform.generation") },
   { key: "family_name", label: t("platform.family") },
 ];
 
+const isEditable = ref(false);
+
+function toggleEditable() {
+  isEditable.value = !isEditable.value;
+}
+
+function updatePlatformName(newName: string) {
+  if (currentPlatform.value) {
+    currentPlatform.value.custom_name = newName;
+    platformApi.updatePlatform({
+      platform: {
+        ...currentPlatform.value,
```

Comment:
you dont have to send all the fields anymore right, just the id?

Reply:
The api method expects the platform object, so we need to send them with the updated properties